lib/src/components/NxTable/__tests__/NxTable.test.tsx

```diff
@@ -55,25 +56,30 @@ describe('NxTable', function() {
       </NxTable>
     );
 
-    expect(component.find('thead th').length).toBe(2);
-    expect(component.find('thead td')).not.toExist();
-    expect(component.find('tbody td').length).toBe(2);
-    expect(component.find('tbody th')).not.toExist();
+    const table = screen.getByRole('table');
+    const rowGroups = within(table).getAllByRole('rowgroup');
+    const headerRows = within(rowGroups[0]).getAllByRole('row');
+    const headerColumnHeaders = within(headerRows[0]).getAllByRole('columnheader');
+    const headerCells = within(headerRows[0]).queryAllByRole('cell');
+    const bodyRows = within(rowGroups[1]).getAllByRole('row');
+    const bodyColumnHeaders = within(bodyRows[0]).queryAllByRole('columnheader');
+    const bodyCells = within(bodyRows[0]).getAllByRole('cell');
 
-    expect(component.find('thead tr')).toHaveClassName('nx-table-row--header');
-    expect(component.find('thead tr')).toHaveClassName('nx-table-row');
+    expect(table).toBeInTheDocument();
+    expect(rowGroups.length).toBe(2);
```

Comment:
There is a `toHaveLength` matcher

Reply:
> There is a `toHaveLength` matcher

Thanks! I updated it here: 3a12e15fa0ec481d388ebca0c0d86293341b0ddf